docs/markdown/Python/python-goals/python-test-goal.md

```diff
@@ -354,6 +354,13 @@ If you use multiple files with `test --debug`, they will run sequentially rather
 >
 > Run your test with `./pants test --debug` as usual.
 
+Failures to collect tests
```

Comment:
Maybe move this lower in the file? Since it's a debugging topic. Leave our other topics higher up like how to use Pants mechanisms.

Reply:
Fair enough, moved to the end of the file. I thought first it makes sense to keep it close to the `--debug` section above, but perhaps indeed it would be better placed in the end.